### DISCUSSION TITLE
What will or should it cost to mint an NFT on Bitclout?

### DISCUSSION THREAD

**Discussion by omicreativedev** [deso-protocol/dips#57] [Q&A]
Burning Question: What will it cost, or SHOULD it cost, to mint an NFT on Bitclout?

**Comment by AikonMishima** [comment]
An important question indeed, especially when we’ll be at 50k plus users minting NFT’s.
It’ll cause congestion and we all know what happens then.

If you search for “fee” in these files, you’ll stumble upon the min max for the fee rate per kb. 
https://github.com/bitclout/core/pull/41/files

An ideal number is hard to set, as it will vary with the number of NFT’s being minted (causing congestion, which requires fee-throttling).

![203D73EF-C10E-4FAB-85DF-B97CDD7F29EE](https://user-images.githubusercontent.com/84419699/125534067-fe1defc7-3812-4b8f-9444-a0d8305f89e3.jpeg)

**Reply by sull** [reply]
it should be same/similar to any post.

**Reply by AikonMishima** [reply]
The Nanofees are used to reduce congestion/over usage.

In other words: if too many people start NFT’ing everything (>100 nft’s per account per day), with minimal $CLOUT loss to fees, data storage will grow wildly.

That’s why I prefer to vote for a throtling mechanism, as it will deter people to mass-NFT every single thing they see.

Not the popular vote, but it will make sense in the future.

**Reply by sull** [reply]
Yeah I expect a flood of NFTs initially. It should eventually level out more but there will always be NFT spam since it's just as easy as making a regular post. 

Not disagreeing but my point is that what applies for a regular post should be about same for NFT. Discouraging basic garbage posts with fees is just as relevant, not just NFT post types.

**Comment by tijno** [comment]
This may help the discussion

Here are the actual transaction fees (in Nanos) per transaction type, Average, min and max.

![20210714-5LM3CNln](https://user-images.githubusercontent.com/69529928/125626289-165f1ec0-6efa-4599-80b5-1f4d526f40c1.png)
